crates/miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -51,6 +51,26 @@ proc.authenticate_account_origin
     # => [storage_offset, storage_size]
 end
 
+#! Asserts that the invocation of a kernel procedure originates from the authentication procedure of
+#! the account.
+#!
+#! Inputs:  []
+#! Outputs: []
+#!
+#! Panics if:
+#! - the invocation of the kernel procedure does not originate from the authentication procedure of the account.
```

Comment:
nit: comment wrapping.

Reply:
wrapped in [improve comment wrapping in api.masm](https://github.com/0xMiden/miden-base/pull/1480/commits/4378659af5407253a8f6fdf73d1fda6aac1a0845)